docs/docs/07-forms.md

```diff
@@ -35,7 +35,7 @@ Like all DOM events, the `onChange` prop is supported on all native components a
 
 ## Controlled Components
 
-An `<input>` with `value` set is a *controlled* component. In a controlled `<input>`, the value of the rendered element will always reflect the `value` prop. For example:
+An `<input>` with a `value` prop is a *controlled* component:
```

Comment:
I disagree.

The first sentence sets out a condition ("Something is X because it has Y."). User acknowledges that and proceed to read the implication ("It will render Y.").

These very paragraphs is what made me confused when I read the docs the first time:

> An `<input>` with `value` set is a *controlled* component. In a controlled `<input>`, the value of the rendered element will always reflect the `value` prop.

> An `<input>` that does not supply a `value` (or sets it to `null`) is an *uncontrolled* component. In an uncontrolled `<input>`, the value of the rendered element will reflect the user's input.

The key difference that makes input controlled or not controlled is weather it has a value property set, not weather the value reflects user input, which is how I understood the difference the first time I read it.